### PR TITLE
INTRO section completed!

### PR DESCRIPTION
**Pull request by adelacecilia**

### CODE REVIEW

**Review by MariaTeo — COMMENTED** [review]
Great start! Keep it up 👌🏼

**Review by samuray90 — APPROVED** [review]
🔟 cool!

**Review by andreimaff — APPROVED** [review]
looks nice 👍🏼

**Review by RalucaRbc — APPROVED** [review]
Looks nice and clean overall 💯 (You may need just a new empty line at the end of each file, but you're doing great so far anyway ❤️ )

File: src/_component/about-me/about-me.pug [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+.column(span="row")
+  h1.about-me__name=data['about-me'].name
+.column(span="row")
+  h2.about-me__position=data['about-me'].position
+.column(span="row" height='10px')
+.column(span="row")
+  h3.about-me__info=data['about-me'].info
+.column(span="row")
+  .info-box__with-shadow
+    p.description=data['about-me'].descriptionList[0]
```
Comment (marianzburlea) [inline src/_component/about-me/about-me.pug]: loop the `data['about-me'].descriptionList` collection using `each`
ask for help if you don't know how to do it

**Review by marianzburlea — CHANGES_REQUESTED** [review]
small changes

**Review by marianzburlea — APPROVED** [review]
:100

**Pull request merged**